charts/launch-agent/templates/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: {{ .Values.namespace }}
   annotations:
     checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
-    checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+    checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
is this just a side bugfix?

Reply:
Correct 😆 
Last time, I duplicated the line to restart the pod when updating the secret, but I kept the same key, which made no difference when changing the secret configuration.
Thanks @wandb-zacharyblasczyk for the heads-up.

Reply:
Awesome